### PR TITLE
feat: cpu training ux [DET-5663, DET-5664]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

- conditional display of aux container count
- separate overall slot allocation bar per resource type
- resource type agnostic resource cards
- promote slotsPerAgent to featured rp properties
- notebook launch ux: remove the resource type picker row

## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/pages/Cluster/ClusterOverview.tsx [hunk 1]
```diff
@@ -171,17 +180,30 @@ const ClusterOverview: React.FC = () => {
...
+            size={ShirtSize.enormous}
+            title={`Compute (${ResourceType.GPU})`}
+            totalSlots={overview.GPU.total} />
+        }
+        {overview.CPU.total > 0 &&
+          <SlotAllocationBar
+            resourceStates={cpuSlotStates}
+            showLegends
+            size={ShirtSize.enormous}
+            title={ResourceType.CPU}
```
Comment (ioga) [inline webui/react/src/pages/Cluster/ClusterOverview.tsx:204]: should that be the similar to line 196, `Compute (${ResourceType.CPU})`?

Comment (hamidzr) [reply webui/react/src/pages/Cluster/ClusterOverview.tsx:204]: aah yeah good catch. I'm working on a fixture (stubbed API response) to showcase these changes as we talked about.

**Comment by hamidzr** [comment]
I can't upload screenshots to the main post for some reason

**Review by apizzini — APPROVED** [review]
LGTM

**Pull request merged**